edns.go

```diff
@@ -577,14 +577,17 @@ func (e *EDNS0_N3U) copy() EDNS0 { return &EDNS0_N3U{e.Code, e.AlgCode} }
 type EDNS0_EXPIRE struct {
 	Code   uint16 // Always EDNS0EXPIRE
 	Expire uint32
+	Empty  bool
```

Comment:
This should have documentation. Something along the lines, Empty is used to signal an empty Expire option in a backwards compatible way, it's not used on the wire.

Reply:
added the comment as suggested